Settings/ViewController.swift

```diff
@@ -1,19 +1,110 @@
+                    ("Экран \"Домой\"", nil, "tv", .disclosureIndicator, false),
+                    ("Универсальный доступ", nil, "folder.badge.person.crop", .disclosureIndicator, false),
+                    ("Обои", nil, "photo", .disclosureIndicator, false),
+                    ("Siri и поиск", nil, "magnifyingglass", .disclosureIndicator, false),
+                    ("Face ID и код-пароль", nil, "faceid", .disclosureIndicator, false)]]
 
+    lazy var tableView: UITableView = {
+        let tableView = UITableView(frame: view.bounds, style: .grouped)
+        tableView.register(UITableViewCell.self, forCellReuseIdentifier: "cell")
+        tableView.delegate = self
+        tableView.dataSource = self
+        tableView.translatesAutoresizingMaskIntoConstraints = false
+
+        return tableView
+    }()
+
+    @objc func switchChanged(_ sender : UISwitch?){
+        if let sender = sender {
+            print("""
+                  Нажат переключатель ячейки \(String(describing: sender.accessibilityIdentifier)) \
+                  в положение \(sender.isOn ? "ВКЛ." : "ВЫКЛ.")
+                  """)
+        }
+    }
 }
 
+extension ViewController: UITableViewDelegate, UITableViewDataSource {
+
+//    MARK: Delegate
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        tableView.deselectRow(at: indexPath, animated: true)
+
+        let dataContent = bigData[indexPath.section][indexPath.row]
+        print("Нажата ячейка \(dataContent.name)")
+    }
+
+//    MARK: DataSource
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return bigData.count
+    }
+
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return bigData[section].count
+    }
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "cell", for: indexPath)
+        var content = cell.defaultContentConfiguration()
+        let dataContent = bigData[indexPath.section][indexPath.row]
+        content.text = dataContent.name
+
+        // Secondary text
+        content.prefersSideBySideTextAndSecondaryText = true
+        content.secondaryTextProperties.font = .systemFont(ofSize: 18)
+        content.secondaryTextProperties.color = .gray
+        content.secondaryText = bigData[indexPath.section][indexPath.row].secondaryText
+
+        // Image
+        content.image = UIImage(systemName: bigData[indexPath.section][indexPath.row].systemImage)
+
+        cell.accessoryType = dataContent.accesoryType
+        cell.contentConfiguration = content
+
+        let switchView = UISwitch(frame: .zero)
+        switchView.setOn(false, animated: true)
+        switchView.accessibilityIdentifier = dataContent.name
+        switchView.addTarget(self, action: #selector(self.switchChanged(_:)), for: .valueChanged)
```

Comment:
Здесь не обязательно так сложно указывать функцию-обработчик, можно вот так:
```suggestion
        switchView.addTarget(self, action: #selector(switchChanged), for: .valueChanged)
```
`self` желательно использовать только тогда, когда это действительно нужно
а параметры функции не обязательно перечислять